web/v2/player/index.ts

```diff
@@ -124,7 +107,12 @@ export interface APIPlayerCompanyHistory {
  * Information about a TruckersMP player.
  * @see https://truckersmp.com/developers/api#operation/get-player-id
  */
-export interface APIPlayer {
+export type APIPlayer = _Base & _APIPlayerVTCHistory & _APIPlayerBanData;
+
+// #region APIPlayer
+
+/** Base interface containing property annotations */
+interface _Base {
```

Comment:
I just noticed that this is still called `_Base`. Maybe `_APIPlayerBase` is a more suitable name?

Reply:
Must have missed that one when renaming the other one - fixed.